3.10/use-cases.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: default
+---
+# ArangoDB Use Cases
```

Comment:
Should we really add more links? It can be distracting if you are sent all over the place for additional information, and the content we can currently link to is not very suitable for beginners anyway. An exception are use cases for which we have a white paper.